{{cookiecutter.project_name}}/.github/workflows/run-tests.yml

```diff
@@ -77,9 +77,17 @@ jobs:
               }
             ]
           run-name: {{ cookiecutter.project_name}} Integration Test
+          {%- if cookiecutter.include_feature_store == "yes" %}
```

Comment:
Wait why are we enabling `test_mode` by default for feature store?

Reply:
yeah, `test_mode` is not needed, the payload for feature store ones should be empty

Reply:
Wait why do we need this at all? Can we revert?

Reply:
It looks weird to just have notebook params be empty. Can we rewrite this so the params are only added if it doesn't include feature store?

Reply:
Because if [cookiecutter.include_feature_store is true, we set the local-notebook-path to be `otebooks/TrainWithFeatureStore.py`](https://github.com/databricks/mlops-stack/blob/3f9251ce8c22379852405f5d55dc6716dd7c71a8/%7B%7Bcookiecutter.project_name%7D%7D/.github/workflows/run-tests.yml#L52), which does not take `env` nor `test_mode`?

Reply:
cool

> It looks weird to just have notebook params be empty. Can we rewrite this so the params are only added if it doesn't include feature store?

cool